src/scss/base/_base.scss

```diff
@@ -2,4 +2,7 @@
   width: 100%;
   max-width: 1100px;
   margin: 0 auto;
+  @media only screen and (max-width: 1200px) {
+    width: 90%;
+  }
```

Comment:
you can use variables for the media querys.
http://thesassway.com/intermediate/responsive-web-design-in-sass-using-media-queries-in-sass-32
here is a link for that